dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -513,7 +521,29 @@ def fetch_new_reports(params, logger=None):
         )
 
     for key, df in ret.copy().items():
-        (geo, sig, _) = key
+        (geo, sig, prop) = key
+
+        if sig == "positivity":
+            # Combine with test volume using publish date.
+            total_key = (geo, "total", prop)
+            ret[key] = unify_testing_sigs(
+                df, ret[total_key]
+            ).drop(
+                "publish_date", axis=1
+            )
+
+            # No longer need "total" signal.
+            del ret[total_key]
+        elif sig == "total":
+            # If the signal is test volume, keep publish date to be used when
+            # combining with test positivity.
+            continue
+        else:
```

Comment:
I was momentarily confused here on whether we're assuming a particular traversal order -- check my understanding?
* we're iterating over a copy, so we'll always encounter both naat `sig` keys no matter what
* we don't actually care which key we see first; the `total` case is only here to prevent it from falling into the `.drop` handling in the else case

If that's correct, I have a slight preference for the following, which makes the latter explicit:

```suggestion
        elif sig != "total":
```

Reply:
Yep, that's the desired logic.